packages/react-components/src/components/SendBox.tsx

```diff
@@ -128,6 +132,8 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
 
   const sendTextFieldRef = React.useRef<ITextField>(null);
 
+  useEffect(() => sendTextFieldRef.current?.focus(), []);
```

Comment:
A little concerned we blindly steal focus when this component mounts (components should be dumb, very generic components). Perhaps we should add a `focusOnMount` property or `focus` method`, then when the chat-composite can use those - don't block PR on this comment but would like to hear what you and others think?

Reply:
good point. 
Focus should be on this SendBox when integrated in Chat for our case, but indeed, Contoso might want to have focus elsewhere when using this component on its own

Reply:
So do we think maybe changing it to a focus method then? or do we think the prop is the better way? im leaning towards the function more myself

Reply:
So I tried just removing the useEffect focus method from this and the narrator completely freaks out. so something I am wondering is are we wanting to take this out or give the option to override the focus? I'm just wondering because wont it go against the accessibility standards we need to meet with what we are developing if we make something that the narrator just freaks on? thoughts here?

Reply:
what about adding a props 'shouldFocusOnMount: boolean;' and then use it like this

useEffect(() => {
  if (shouldFocusOnMount) {
    sendTextFieldRef.current?.focus();
  }
}, []);


and set it to true in the composite when calling Sendbox

Reply:
yep, good point.
So in composite it will be a prop like 'shouldFocusOnSendBoxonMount' or something like that

Reply:
I think at the composite level it could be the same api: `focusOnMount: boolean` or `focus: () => void` (and the chat composite logic can dictate that the sendbox is the thing that gets focus when called/mounted)